cves/CVE-2014-0483.yml

```diff
@@ -6,7 +6,6 @@ yaml_instructions: |
   For one-line strings, you can just use quotes after the colon
   For multi-line strings, as we do for our instructions, you put a | and then
```

Comment:
I think you accidentally edited the wrong file here...

Reply:
My apologies, i wasnt able to say i switched files from 2009-3965 to This file based on the fact that the original CVE that was assigned to me  had a lot of pending information so i switched to something that had a bit more info available. (this was approved by the professor)